src/models/transfers.js

```diff
@@ -29,6 +29,8 @@ const Condition = require('five-bells-condition').Condition
 
 const validExecutionStates = ['prepared']
 const validCancellationStates = ['prepared', 'proposed']
+const SIGNATURE_TYPE_ED25519 = 'ed25519-sha512'
+const SIGNATURE_TYPE_SHA256 = 'sha256'
```

Comment:
I think this is going to be confusing to people because "sha256" is a hashing algorithm, which is not a signature scheme. I assume we are using it like a signature here, but does it actually satisfy the technical definition of a digital signature? If so, I'm guessing it has a more accurate name than just the name of the hashing algorithm.